tools/lisa-build-asset

```diff
@@ -60,22 +70,54 @@ def list_assets(recipe_dir):
 def get_asset_recipe(asset, recipe_dir):
     return os.path.join(recipe_dir, '{}.recipe'.format(asset))
 
-def make(asset, actions, arch, toolchain, recipe_dir, build_dir):
+def get_recipe_name(asset):
+    return asset + ".recipe"
+
+def make_native_build(asset, arch, recipe_dir, build_dir, alpine_source_dir, arch_chroot_dir, env):
+    # Install the chroot
+    subprocess.check_call(['sudo', os.path.join(alpine_source_dir, 'alpine-chroot-install'),
```

Comment:
How long does that take (roughly) ? If it's more than let's say 20s we probably want to factor out the setup and only do it once for each arch, rather than each arch x each recipe.

Reply:
It takes ~15s.
Another thing is that the alpine script sets the env variables and Alpine packages required for an asset. Sharing a container is possible, but would mean re-writing parts of the alpine script instead of just passing the right parameters to the script.